.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,203 @@
+name: Nightly
```

Comment:
nit: `Nightly Build` or similar might make sense since I anticipate us creating more actions in the near future, the specificity is probably useful

Reply:
Good call.